### PR TITLE
Gradle publish, checkstyle, classification

### PR DESCRIPTION
**Pull request by dboulware**
This PR:
- adds a gradle publish task to facilitate publishing the jar as a GitHub package
- adds google checkstyle and reformats code to comply with checkstyle
- renames the beamwidth related java variables to be consistent with beamwidth being one word
- adds a required classification field to the measurement object.

### CODE REVIEW

File: ntia-core.sigmf-ext.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ The Measurement object summarizes the basic measurement information including  w
 |`frequency_tuned_high`|true|double|Hz|Highest tuned  frequency.|
 |`frequency_tuned_step`|false|double|Hz|Step between tuned frequencies of a `"scan"` measurement. Either `frequency_tuned_step` or `frequencies_tuned` SHOULD be included for `"scan"` measurements.|
 |`frequencies_tuned`|false|double[]|Hz|Array of tuned frequencies of a `"scan"` measurement. Either `frequency_tuned_step` or `frequencies_tuned` SHOULD be included for `"scan"` measurements.|
+|`classification`|true|string|N/A|The classification markings for the acquisition.|
```
Comment (MichaelCotton) [inline ntia-core.sigmf-ext.md:28]: Remove leading "The". Append with "Either `UNCLASSIFIED`, `SENSITIVE`, `CLASSIFIED`.

**Comment by dboulware** [comment]
I think these have all been remedied.

File: java/src/main/java/gov/doc/ntia/sigmf/ext/global/waveform/Ieee80211p.java [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+  protected Double shortInterFrameSpace;
+
+  @JsonProperty(value = "preamble_frame", required = false)
+  protected Integer[] preambleFrame;
+
+  @JsonProperty(value = "number_of_info_bits", required = false)
+  protected Long numberOfInfoBits;
+
+  @JsonProperty(value = "signal_to_noise_ratio", required = false)
+  protected Double signalToNoiseRation;
```
[the remark this answers is not in the document]
Comment (jhazentia) [reply java/src/main/java/gov/doc/ntia/sigmf/ext/global/waveform/Ieee80211p.java:49]: remove trailing n in member variable

File: java/src/main/java/gov/doc/ntia/sigmf/examples/callibration/CalibrationAnnotationExample.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+import gov.doc.ntia.sigmf.Annotation;
+import gov.doc.ntia.sigmf.Global;
+import gov.doc.ntia.sigmf.MetaDoc;
+import gov.doc.ntia.sigmf.examples.Example;
+import gov.doc.ntia.sigmf.ext.annotation.sensor.CalibrationAnnotation;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CalibrationAnnotationExample implements Example {
```
Comment (jhazentia) [inline java/src/main/java/gov/doc/ntia/sigmf/examples/callibration/CalibrationAnnotationExample.java:11]: extra "l" in folder name: callibration -> calibration

File: java/src/main/java/gov/doc/ntia/sigmf/examples/ExampleGenerator.java [hunk 4]
```diff
@@ -9,107 +9,93 @@
...
-        DigitalFilterAnnotationExample digitalFilterAnnotationExample = new DigitalFilterAnnotationExample();
-        save("digitalFilterAnnotation.json", digitalFilterAnnotationExample.getExample());
+    DigitalFilterAnnotationExample digitalFilterAnnotationExample =
+        new DigitalFilterAnnotationExample();
+    save("digitalFilterAnnotation.json", digitalFilterAnnotationExample.getExample());
 
-        CallibrationAnnotationExample cal = new CallibrationAnnotationExample();
-        save("callibration.json", cal.getExample());
+    CalibrationAnnotationExample cal = new CalibrationAnnotationExample();
+    save("callibration.json", cal.getExample());
```
Comment (jhazentia) [inline java/src/main/java/gov/doc/ntia/sigmf/examples/ExampleGenerator.java:54]: extra "l" in file name: "callibration.json" -> "calibration.json"

File: java/src/main/java/gov/doc/ntia/sigmf/ext/global/waveform/Ieee80211p.java [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+
+  public Long getNumberOfInfoBits() {
+    return numberOfInfoBits;
+  }
+
+  public void setNumberOfInfoBits(Long numberOfInfoBits) {
+    this.numberOfInfoBits = numberOfInfoBits;
+  }
+
+  public Double getSignalToNoiseRation() {
```
[the remark this answers is not in the document]
Comment (jhazentia) [reply java/src/main/java/gov/doc/ntia/sigmf/ext/global/waveform/Ieee80211p.java:149]: remove trailing n in getter and setter name

**Pull request merged**